packages/rancher-vsphere/rancher-vsphere-csi/generated-changes/patch/templates/node/daemonset.yaml.patch

```diff
@@ -0,0 +1,335 @@
+--- charts-original/templates/node/daemonset.yaml
```

Comment:
Seems like what was committed in https://github.com/rancher/charts/blob/dev-v2.6/charts/rancher-vsphere-csi/100.1.1%2Bup2.3.0-rancher1/templates/node/daemonset.yaml and https://github.com/rancher/vsphere-charts/blob/main/charts/rancher-vsphere-csi/templates/node/daemonset.yaml fundamentally differ from each other...

Based on the fact that the code committed in rancher/charts has [`nodeSelector`, `tolerations`, and `affinity` in entirely the wrong location under `.spec`](https://github.com/rancher/charts/blob/53824f30f4d8bf95339ec310662cadaae2f98869/charts/rancher-vsphere-csi/100.1.1%2Bup2.3.0-rancher1/templates/node/daemonset.yaml#L12-L48), I'm inclined to believe whatever is in upstream is more accurate. 

So using upstream instead of what was committed in rancher/charts looks good to me.